libs/ledger-live-common/scripts/sync-families-dispatch.mjs

```diff
@@ -82,27 +82,42 @@ function genCoinFrameworkTarget(targetFile) {
     switch (targetFile) {
       case "bridge/js.ts":
         imports += `import { bridge as ${family} } from "../../families/${family}/setup";\n`;
+        exprts += `\n  ${family},`;
```

Comment:
Why those export update do not generate a diff in generated files ? is it supposed to ?

Reply:
Not really. If there is a modification in generated directory, that means we can face some breaking dependency in LLC, LLD and LLM file.